hackathon_site/dashboard/frontend/src/pages/Orders/Orders.js

```diff
@@ -2,14 +2,15 @@ import React from "react";
 // import styles from "./Orders.module.scss";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
-
+import TeamModal from "../../components/dashboard/EditTeamModal/TeamModal";
 const Orders = () => {
     return (
-        <>
+        <div>
             <Header />
             <Typography variant="h1">Orders</Typography>
-            <p>IEEEEEE</p>
```

Comment:
@Leo6Leo the test is failing because you changed the content on this page. Since the Edit Team Modal will be on the dashboard page you don't need to change this file.